src/Classes/Utils/EventListeners.ts

```diff
@@ -30,7 +30,7 @@ export class EventListeners {
         /* events target element */
         if (options.interactivity.detectsOn === InteractivityDetect.window) {
             container.interactivity.element = window;
-        } else if (container.options.interactivity.detectsOn === InteractivityDetect.parent) {
+        } else if (container.options.interactivity.detectsOn === InteractivityDetect.parent && container.canvas.element) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)